api/config.py

```diff
@@ -224,8 +224,7 @@ class Configuration(CoreConfiguration):
             "key": Announcements.SETTING_NAME,
             "label": _("Scheduled announcements"),
             "description": _("Announcements will be displayed to authenticated patrons."),
-            "category": "Basic Information",
-            "type": "list",
+            "category": "Announcements",
             "format": "announcements"
```

Comment:
Just recording the decision we made in our private conversation: you'll change this from format="announcements" to type="announcements". When we redo the configuration system, we'll use more specific terms like "ui_type" and "data_format".